package.json

```diff
@@ -449,6 +449,11 @@
           "description": "Override the 'ignorecase' option if the search pattern contains upper case characters.",
           "default": true
         },
+        "vim.camelCaseMotion": {
```

Comment:
I'm trying to make this more hierarchic. Can you make this `vim.camelCaseMotion.enable` instead? Such that if we do end up creating more configs for camel case, they can live under `vim.camelCaseMotion`

Reply:
Good idea! I've switched the setting over as you suggested and pushed the new changes.